x-pack/plugins/security_solution/common/endpoint/service/artifacts/constants.ts

```diff
@@ -5,6 +5,18 @@
  * 2.0.
  */
 
+import {
+  ENDPOINT_TRUSTED_APPS_LIST_ID,
+  ENDPOINT_EVENT_FILTERS_LIST_ID,
+  ENDPOINT_HOST_ISOLATION_EXCEPTIONS_LIST_ID,
+} from '@kbn/securitysolution-list-constants';
+
 export const BY_POLICY_ARTIFACT_TAG_PREFIX = 'policy:';
 
 export const GLOBAL_ARTIFACT_TAG = `${BY_POLICY_ARTIFACT_TAG_PREFIX}all`;
+
+export const ALL_ENDPOINT_ARTIFACT_LIST_IDS: readonly string[] = [
```

Comment:
Thinking out loud: We can use this here too: https://github.com/elastic/kibana/blob/fcb1b9d08a0b02d8a3709f770549702a43454ebd/x-pack/plugins/security_solution/server/fleet_integration/handlers/remove_policy_from_artifacts.ts#L18

Reply:
@dasansol92 Yeah, that would be ideal. I also been thinking that this array should exists in `@kbn/securitysolution-list-constants` so that we don't have to keep our own copy, which we will need to remember to update every time a new artifact is added. I'm going to open an issue for tracking